src/ParseModelDecorator/index.ts

```diff
@@ -0,0 +1,138 @@
+import 'reflect-metadata';
+import type { SchemaDefinition } from '@/rule/RuleElementCreator';
+
+export interface GridOptions {
+  canView: boolean;
+  canRead: boolean;
+  canEdit: boolean;
+  canFilter: boolean;
+  canSort: boolean;
+}
+export const defaultConfig: GridOptions = {
```

Comment:
It's probably a good idea to add a `Readonly` here to avoid accidental modification